casts/cards/src/UICollectionCard/UICollectionCard.tsx

```diff
@@ -73,7 +73,7 @@ export function UICollectionCard({
                             </UILabel>
                         ) : null}
                     </View>
-                    <Badge badge={badge} />
+                    <Badge badge={badge} testID={`collection_badge_${title}${typeof badge === 'string' ? '_balance_' + badge :  ''}`} />
```

Comment:
I don't like seeing `balance` here... since it might be not balance, but smth else.
```suggestion
                    <Badge badge={badge} testID={`collection_badge_${title}:${badge.toString()}`} />
```

Why not like this?

Reply:
If `badge` is a string - we render the balance of the collection, otherwise, we render the link icon, which I don't need here.

Reply:
Maybe call it `count` then?

Reply:
Done @savelichalex